--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -70,4 +70,8 @@
 ###############################################################################
 bazel_dep(name = "score_tooling", version = "1.1.2")
 bazel_dep(name = "score_docs_as_code", version = "4.0.3")
+local_path_override(
+    module_name = "score_docs_as_code",
+    path = "../docs-as-code",
+)
 bazel_dep(name = "score_process", version = "1.5.4")
